corehq/apps/domain_migration_flags/api.py

```diff
@@ -41,7 +41,7 @@ def get_migration_complete(domain, slug):
     return get_migration_status(domain, slug) == MigrationStatus.COMPLETE
 
 
-@quickcache(['domain', 'slug'], skip_arg='strict')
+@quickcache(['domain', 'slug'], skip_arg='strict', timeout=60 * 60)
```

Comment:
Is it also worth setting the `memoized_timeout` to longer (defaults to 10 seconds)?